tests/test_vxlan_vtep_vni.rb

```diff
@@ -210,4 +214,23 @@ def test_suppress_arp
     vni.suppress_arp = vni.default_suppress_arp
     refute(vni.suppress_arp, 'suppress_arp should be disabled')
   end
+
+  def test_suppress_uuc
+    skip('Platform does not support suppress-unknown-unicast') unless
+    node.cmd_ref.supports?('vxlan_vtep_vni', 'suppress_uuc')
```

Comment:
I tried this and I'm not seeing any unsupported errors on unsupported platforms. The skips work correctly on all platforms. Can we stick with my code till I identify the reason for this?